tensorflow/core/grappler/utils/topological_sort.cc

```diff
@@ -96,8 +96,8 @@ Status ComputeTopologicalOrder(
                  "at node = "
               << graph.node(back).DebugString();
       for (int i = 0; i < graph_view.num_nodes(); ++i) {
-        if (num_ready_inputs[i] !=
-            static_cast<int>(graph_view.GetFanin(i).size())) {
+        const int graph_view_Get_fanin_i_size = graph_view.GetFanin(i).size();
+        if (num_ready_inputs[i] != graph_view_Get_fanin_i_size) {
```

Comment:
Still has a capital letter. Why not use the shorter name `max_size`? It's a local variable, defined just above, not used anywhere else, so it does not harm readability.

And then also change the other local above (line 84) to be `max_size` too.

Reply:
Made the change.